docs/config.md

```diff
@@ -227,6 +227,27 @@ cpu_only_composing_models: <comma-delimited-string-list>
 # Enables the searching of request rate (instead of concurrency)
 [ request_rate_search_enable: <bool> | default: false]
 
+# Minimum percentage of the search space to profile when using Optuna
+[ min_percentage_of_search_space: <int> | default: 5]
+
+# Maximum percentage of the search space to profile when using Optuna
+[ max_percentage_of_search_space: <int> | default: 10]
+
+# Minimum number of trials to profile when using Optuna
+[ optuna_min_trials: <int> | default: 20]
```

Comment:
One of [percentage vs trials] has precedence, right? And the other is only used if it is specified? If so, is there any point in specifying a default?  Reading the docs it isn't clear to me what will be searched by default

Reply:
Right now I made percentage have precedence. So, no there's not really a point of specifying a default for the other.